vanilla-framework/vanilla-framework
# Update docs/color page content

## Done

- Added accessibility section
- Added imagery
- Added do's and don't examples
- Check content matches copy doc [here](https://docs.google.com/document/d/1Ly43Jb5Cav3Eee0FgNUVTXSJAjB71KNMqLWHf9c5scw/edit#)

## QA

- Pull code
- Run `./run serve --watch`
- Check https://docs.vanillaframework.io/en/settings/color-settings

## Details
- Fixes https://github.com/ubuntudesign/vanilla-design/issues/339
